app/controllers/audios_controller.rb

```diff
@@ -20,6 +20,7 @@ def new
     @formdef = PSSBrowserUploads.av_form_definition('audio')
     @accepted_types = %w(.mp3 .mp2 .mp4a .wav .flac .aif .aiff .wma .mpga .oga
                          .ogg .au .adp .aac .weba).join(',')
+    @source = Source.find_by_id(params[:source_id])
```

Comment:
I instantiated the `Source` for two reasons.  First, as you noted, it works as check to verify that the ID exists.  The checks are in the views themselves.  See for instance: https://github.com/dpla/primary-source-sets/pull/82/files#diff-87342b07e87a296b812ab3e844a147efR6  But let me know if you think there's a better way to do the check.

Second, the views use other attributes of the `Source`, not just the ID.  See for instance: https://github.com/dpla/primary-source-sets/pull/82/files#diff-87342b07e87a296b812ab3e844a147efR16